xml/ay_bootloader.xml

```diff
@@ -260,6 +220,61 @@
      </para>
     </listitem>
    </varlistentry>
+   <varlistentry>
+    <term>generic_mbr</term>
+    <listitem>
+     <para>
+      Write generic boot code to the MBR (will be ignored if
+      <literal>boot_mbr</literal> is set to <literal>true</literal>).
+     </para>
+<screen>&lt;generic_mbr config:type="boolean"&gt;false&lt;/generic_mbr&gt;</screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>gfxmode</term>
+    <listitem>
+     <para>
+      Graphical resolution of the &grub; screen (requires &lt;terminal&gt; to
+      be set to <literal>gfxterm</literal>).
+     </para>
+     <para>
+      Valid entries are <literal>auto</literal>, <literal><replaceable>
+      HORIZONTAL</replaceable>x<replaceable>VERTICAL</replaceable></literal>,
+      or <literal><replaceable>HORIZONTAL</replaceable>x<replaceable>VERTICAL
+      </replaceable>x<replaceable>COLOR DEPTH</replaceable></literal>. You can
+      see the screen resolutions supported by &grub; on a particular system by
+      using the <command>vbeinfo</command> command at the &grub; command line
+      in the running system.
+     </para>
+<screen>&lt;gfxmode&gt;1280x1024x24&lt;/gfxmode&gt;</screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>os_prober</term>
+    <listitem>
+     <para>
+      If set to <literal>true</literal>, automatically searches for operating
+      systems already installed and generates boot entries for them during the
+      installation.
+     </para>
+<screen>&lt;os_prober&gt;false&lt;/os_prober&gt;</screen>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>password</term>
+    <listitem>
+     <para>
+      If defined, then it protects boot via password. It has three subelements.
+      <literal>value</literal> holds password value. It can be either plain text
+      or already encrypted one. If <literal>encrypted</literal> is set to <literal>
+      true</literal> it means that <literal>value</literal> is already encrypted
+      password. <literal>unrestricted</literal> is to set unrestricted option for
+      password protection. For more details see password protection section of
+      booting manual TODO: how to reference section from grub2.xml?
```

Comment:
Hi, I just need help here. I am not much familiar with format. So is there way to add here link to grub2.xml section about password protected boot?

Reply:
There is! Like this example:

`<phrase>For more information on  boot passwords, see <xref linkend="sec-grub2-password"/>.</phrase>`

Use the appropriate XML ID. It is not possible to create links between different books. The `<phrase/> ` tag creates a link when linking is possible, and when a link is not possible, it looks like this:

> For more information on boot passwords, see _Book “Administration Guide”, Chapter 14 “The boot loader GRUB 2”, Section 14.2.6 “Setting a boot password_”.